src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h

```diff
@@ -304,11 +305,10 @@ namespace OpenMS
         f.setQuality(0, quality);
         f.setOverallQuality(quality);
 
-        ConvexHull2D::PointArrayType hull_points;
```

Comment:
We want to be able to capture the `hull_points`.  If this is missing in `PeakIntegrator::integratePeak` we will need to add it in as an option.